src/pages/product/Product.jsx

```diff
@@ -192,22 +221,22 @@ const Product = () => {
             <tbody>
               {currentProducts.map((product, index) => (
                 <tr key={index} className="hover:bg-gray-50">
-                  <td className="px-5 py-2 border-b text-sm text-gray-400 whitespace-nowrap  border-gray-200">
+                  <td className="px-5 border-b text-sm text-gray-400 whitespace-nowrap  border-gray-200">
```

Comment:
Bisa rubah untuk semua tdnya menjadi seperti ini agar lebih mendekati desgin
```suggestion
                  <td className="border-b px-4 py-2 text-xs text-black/60">
```